code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -706,3 +706,45 @@
 /obj/structure/closet/crate/extinguisher_catridges/fill()
 	for(var/a = 1 to 12)
 		new /obj/item/weapon/reagent_containers/extinguisher_refill(src)
+
+/obj/structure/closet/crate/autakh
+	name = "aut'akh crate"
+	desc = "Contains a number of limbs and augmentations created by the Aut'akh Commune."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "autakh_crate"
+	icon_opened = "autakh_crateopen"
+	icon_closed = "autakh_crate"
+
+/obj/structure/closet/crate/autakh/fill()

```

Comment:
Reduce that to two of each limb and one of each grasper.
Anything else that exceeds that should be ordered from cargo

Reply:
fine